packages/grafana-data/src/types/dataFrame.ts

```diff
@@ -50,6 +51,8 @@ export interface FieldConfig {
 
   // Panel Specific Values
   custom?: Record<string, any>;
+
+  scopedVars?: ScopedVars;
```

Comment:
why is scopedVars here?

Reply:
Added those for the overrides processor to be able to interpolate field variables. See https://github.com/grafana/grafana/pull/22047/files#diff-3b41a4dd550fb20fd66edb288940b5c9R17 for example. So particular field override config can specify whether or not variables should be interpolated, hence we need to have access to the vars